tests/zfs-tests/tests/functional/events/events_002_pos.ksh

```diff
@@ -88,8 +92,8 @@ while ! is_pool_scrubbed $MPOOL; do
 	sleep 1
 done
 
-# 5. Start the ZED and verify it only handled the new missed events.
-log_must zed_start
+# 6. Start the ZED and verify it only handled the new missed events.
+log_must zed_start false
 log_must sleep 20
```

Comment:
We observed that a 20 second sleep here isn't always enough.  What would be better if we could rework this while we're here so it slept until say `sysevent.fs.zfs.resilver_finish` was found in the log or say a 60s timeout occurs.

Reply:
Changed to loop until we find the resilver_finish in the zed log or 60 seconds whichever comes first.